pagemmultiscope_spec.rb

```diff
@@ -0,0 +1,75 @@
+require File.dirname(__FILE__) + '/spec_helper'
+require File.expand_path("../../lib/pagem", __FILE__)
+
+describe PagemMultiscope do
+  let(:scope) { double("Scope") }
+  let(:second_scope) { double("Scope") }
+
+  before(:each) do
+    scope.stub(:size).and_return(101)
+    second_scope.stub(:size).and_return(202)
+    scope.stub(:where => scope, :limit => scope, :offset => scope)
+    second_scope.stub(:where => second_scope, :limit => second_scope, :offset => second_scope)
+  end
+
+  it 'inherits from Pagem' do
+    expect(PagemMultiscope).to be < Pagem
+  end
+
+  it 'gets the sizes of the scopes if they are not passed in' do
+    scope.should_receive(:count).and_return(101)
+    @multipager = PagemMultiscope.new(scope, second_scope, {:page => 1})
+    @multipager.instance_eval { @count}.should == 303
+  end
+
+  it 'assigns of the scopes if they are passed in' do
+    @multipager = PagemMultiscope.new(scope, second_scope, {page: 1}, {
+      count_number: 303,
+      first_scope_count: 101,
```

Comment:
I don't think it's necessary for <code>first_scope_count</code> and <code>second_scope_count</code> to be passed in to correctly calculate the count from the <code>count_number</code> param, I think this should be separate specs testing that passing in <code>count_number</code> returns the right count, and another spec testing that passing in <code>first_scope_count</code> and <code>second_scope_count</code> calculates the same correct count